client/hooks/useJob.ts

```diff
@@ -53,7 +53,14 @@ const useJob = ({ category, children }) => {
             ...getPublishedStatusText({ derivatives, ingesting, published }),
         ];
     };
-    const getStatusText = ({ derivatives, minutes_since_upload, documents, audio, ingesting, published }) => {
+    const getStatusText = ({
```

Comment:
Does this still need to be typed? It just seems to have been reformatted so far.

Reply:
Made change.

Reply:
Thanks! Just one syntax question: I see that you defined an interface to solve this, and all the fields of the interface are comma-separated. I thought Typescript interfaces usually had a semi-colon at the end of every line. The linter isn't complaining, but I'm curious if this needs to be changed.

Reply:
Weird. I was using the api for reference. I'll make the change.